pytests/test_config.py

```diff
@@ -6,25 +6,21 @@ def testConfigGet(env):
     config_names = [configs[a].split('.')[1] for a in range(0, len(configs), 2)]
     config_vals = [configs[a] for a in range(1, len(configs), 2)]
     for name in config_names:
-        env.cmd('RG.CONFIG', 'GET', name)
+        env.cmd('RG.CONFIGGET', name)
```

Comment:
In general we will not use RedisGears specific config command because sense Redis 7 we can add module configuration to Redis configuration and use the Redis `config` command. This is for legacy purposes only. But we can change it to contains `_`.